templates/link.hbs

```diff
@@ -0,0 +1 @@
+<li class="header__item"><a  class="header__link" href={{linkDestination}}>{{linkText}}</a></li>
```

Comment:
Links are not centered vertically. Please add `align-items: center;` to the container (`.header__links`).

Reply:
Checking the Content type `link`, and I noticed the field `Link Destination` was a list. I created a new field `url`. please use it instead of `linkDestination`: `href="{{ url }}"`